api-specs/Gateway-EE/latest/kong-ee.yaml

```diff
@@ -14388,6 +14388,140 @@ paths:
       operationId: get-event-hooks-event-hook-id-ping
       description: |
         Ping a webhook event hook.
+  '/{workspace_name_or_id}/rbac/roles':
```

Comment:
Similarly, for consistency with the rest of the spec, should these endpoints be under RBAC instead of under Workspaces? I don't know which one is optimal organization, but currently we do things like `/{workspace}/certificates` under Certificates, and so on.